xbmc/cores/AudioEngine/Sinks/windows/AESinkFactoryWin.cpp

```diff
@@ -143,11 +144,17 @@ const char *WASAPIErrToStr(HRESULT err)
           wfxex.Format.nChannels = 8; // Four IEC 60958 Lines.
           wfxex.dwChannelMask = KSAUDIO_SPEAKER_7POINT1_SURROUND;
           break;
-        case CAEStreamInfo::STREAM_TYPE_DTSHD:
+        case CAEStreamInfo::STREAM_TYPE_DTSHD_MA:
           wfxex.SubFormat = KSDATAFORMAT_SUBTYPE_IEC61937_DTS_HD;
           wfxex.Format.nChannels = 8; // Four IEC 60958 Lines.
           wfxex.dwChannelMask = KSAUDIO_SPEAKER_7POINT1_SURROUND;
           break;
+        case CAEStreamInfo::STREAM_TYPE_DTSHD:
+          wfxex.SubFormat = KSDATAFORMAT_SUBTYPE_IEC61937_DTS_HD;
+          wfxex.Format.nChannels = 2; // One IEC 60958 Lines.
+          wfxex.dwChannelMask = KSAUDIO_SPEAKER_5POINT1;
+          break;
+
```

Comment:
why this empty line?

Reply:
Because I wanted you to spot it in review - there is no reason for that.